dependencies/pom.xml

```diff
@@ -146,6 +146,7 @@
         <version.lib.snakeyaml>2.0</version.lib.snakeyaml>
         <version.lib.typesafe-config>1.4.2</version.lib.typesafe-config>
         <version.lib.tyrus>2.0.4</version.lib.tyrus>
+        <version.lib.vertx-core>4.3.7</version.lib.vertx-core>
```

Comment:
Test dependencies should only be defined in pom.xml in project root

Reply:
thx moved to the correct pom